src/mixins/I13nMixin.js

```diff
@@ -216,6 +216,21 @@ var I13nMixin = {
     },
 
     /**
+     * _shouldFollowLink, provide a hook to check followLink.
+     * It check if component implement its own shouldFollowLink() method, 
+     * otherwise return props.followLink or props.follow
+     * @method _shouldFollowLink
+     * @private
+     */
+    _shouldFollowLink: function () {
```

Comment:
as we discussed, we don't encourage users to use `I13nMixin` directly, (use createI13nNode instead), 
so there's no document for `I13nMixin`, I'm ok with this if we all agree this is clear enough as a document